### PR TITLE
remove  multi-startup

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@Tratcher  please review. View [here](https://review.docs.microsoft.com/en-us/aspnet/core/fundamentals/startup?branch=ra%2Fstartup)

>You can define separate Startup classes for different environments, and the appropriate one will be selected at runtime.
Is that still true?

**Comment by Tratcher** [comment]
hm, it wasn't completely removed, just turned off by default.
If you specify `startupAssembly` in the WebHost config / options then hosting will load that assembly and search for a Startup or Startup[Environment] type
https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNetCore.Hosting/Internal/StartupLoader.cs#L66

File: aspnetcore/fundamentals/startup.md [hunk 1]
```diff
@@ -46,26 +46,22 @@ Additional services, like `IHostingEnvironment` and `ILoggerFactory` may also be
 
 ## The ConfigureServices method
 
-The `Startup` class can include a `ConfigureServices` method that takes an `IServiceCollection` parameter and optionally returns an `IServiceProvider`. The `ConfigureServices` method is called before `Configure`, as some features must be added before they can be wired up to the request pipeline.
+The [ConfigureServices](https://docs.microsoft.com/en-us/aspnet/core/api/microsoft.aspnetcore.hosting.startupbase#Microsoft_AspNetCore_Hosting_StartupBase_ConfigureServices_Microsoft_Extensions_DependencyInjection_IServiceCollection_) method is optional, but must be called before  `Configure`, as some features must be added before they can be wired up to the request pipeline. [Configuration options](configuration.md) are set in this method.
```
Comment (Tratcher) [inline aspnetcore/fundamentals/startup.md]: must -> will

**Comment by Tratcher** [comment]
UseStartup is still the recomend appoach though

**Comment by Rick-Anderson** [comment]
@Tratcher  applied your feedback.

File: aspnetcore/fundamentals/startup.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ The `Startup` class configures the request pipeline that handles all requests ma
...
 
-You can define separate `Startup` classes for different environments, and the appropriate one will be selected at runtime. Learn more in [Working with multiple environments](environments.md#startup-conventions).
+You can define separate `Startup` classes for different environments, and the appropriate one will be selected at runtime. If you specify `startupAssembly` in the WebHost configuration or options, hosting will load that startup assembly and search for a `Startup` or `Startup[Environment]` type. See [FindStartupType](https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNetCore.Hosting/Internal/StartupLoader.cs) in `StartupLoader` and [Working with multiple environments](environments.md#startup-conventions). `UseStartup<TStartup>` is the recommended approach.
```
Comment (Tratcher) [inline aspnetcore/fundamentals/startup.md]: permalink to rel/1.1.0, not dev

File: aspnetcore/fundamentals/startup.md (hunk 1 above)
Comment (Tratcher) [inline aspnetcore/fundamentals/startup.md]: missed this comment.

**Pull request merged**